bin/connex/connex.ml

```diff
@@ -189,8 +189,8 @@ let move base basename =
           if !detail == nb then List.iter (print_family base basename) ifaml
           else print_family base basename ifam;
           (if !statistics then
-           try Hashtbl.replace hts nb (Hashtbl.find hts nb + 1)
-           with Not_found -> Hashtbl.add hts nb 1);
+             try Hashtbl.replace hts nb (Hashtbl.find hts nb + 1)
```

Comment:
can you re-write this to be more clear (no try catch):
```ocaml
match Hashtbl.find_opt hts nb with
| None -> Hashtbl.replace hts nb 1
| Some nb -> Hashtbl.replace hts (nb +1)
```

Reply:
done, but check again. I struggled